### DISCUSSION TITLE
Client image shows empty folders

### DISCUSSION THREAD

**Discussion by decenttp** [ltsp/ltsp#723] [General] [answered]
I recently created a new client image using Ubuntu 22.04. Earlier image was created using 20.04. Users are being authenticated but their home folders seem to be different and they cannot access their files. They do not have this issue when they switch between other images but this one. What could be wrong?

Thanks

**Comment by alkisg** [comment, marked as answer]
How is the image created, chroot, VM, chrootless?
Does it have the ltsp and sshfs packages installed in it?

Also, boot the client, login, type the following commands, and upload their output here:

```
cat /proc/cmdline
mount
id
```

**Reply by decenttp** [reply]
I removed and then re-installed ltsp package and for some reason that resolved the problem. Thanks for the pointer.
